src/Microsoft.AspNet.OData.Shared/Query/Expressions/SelectExpandBinder.cs

```diff
@@ -1057,23 +1045,26 @@ private Expression AddOrderByQueryForSource(Expression source, OrderByClause ord
                     _settings.EnableConstantParameterization);
             }
 
-            if (_settings.PageSize.HasValue || modelBoundPageSize.HasValue || hasTopValue || hasSkipvalue)
+            if (structuredType is IEdmEntityType)
```

Comment:
don't give a variable name for 'is' ?

Reply:
@xuzhg In this case we just need to know we're dealing with an `IEdmEntityType` and not an `IEdmComplexType`. We have no use for the variable.